src/editing.md

```diff
@@ -180,31 +182,82 @@ this is not recommended, for the following reasons:
   trees with thousands of items can actually break the display of
   the deck list in the current implementation.
 
+### Using Tags
+
 Instead of creating lots of little decks, it's a better idea to use tags
-and/or fields to classify your content. Instead of creating a "food
-verbs" decks for example, you could add those cards to your main
-language study deck, and tag the cards with "food" and "verb". Each card
-can have multiple tags, which means you can do things like search for
-all verbs, or all food-related vocabulary, or all verbs that are related
-to food.
+and/or fields to classify your content. Tags are a useful way to boost 
+search results, find specific content, and maintain your collection 
+organized, in addition to (or in combination with) your deck / subdecks 
+structure. There are many ways of using tags and flags effectively, and
+thinking in advance about how you want to use them will help you decide
+what will work best for you.
+
+Some people prefer using decks and subdecks to keep their cards organized,
+but using tags have a big advantage over decks for that: you can add several
+tags to a single note, but a single note can only belong to one deck. This 
+allows using [filtered decks](filtered-decks.md), and makes tags a more 
+powerful and flexible categorization system than decks in most cases. 
+
+For example, instead of creating a "food verbs" deck, you could add those 
+cards to your main language study deck, and tag the cards with "food" and
+"verb". Since each card can have multiple tags, you can do things like 
+[search](searching.md#tags-decks-cards-and-notes) for all verbs, or all 
+food-related vocabulary, or all verbs that are related to food. 
+
+You can add tags from the Edit window and from the [Browser](browsing.md), and you can also add,
+delete, rename, or organize tags hierarchically there. Please, note than
+tags work at [note](getting-started.md#notes--fields) level, which means that when you tag a card that has siblings,
+all the siblings will be tagged as well. If you need to tag a single card,
+but not its siblings, you should consider using flags instead.
+
+### Using Flags
+
+Flags are similar to Tags, but they will appear during study in the review 
```

Comment:
```suggestion
Flags are similar to tags, but they will appear during study in the review 
```
I'm not so sure they're that similar to tags, and it might be worth cutting that part.